package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "awesome_books",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/denisdiaconu/awesome_books.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/denisdiaconu/awesome_books/issues"
+  },
+  "homepage": "https://github.com/denisdiaconu/awesome_books#readme",
+  "devDependencies": {
+    "@babel/core": "^7.14.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.14.0",
+    "@babel/preset-env": "^7.14.2",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.2.2",
+    "css-loader": "^5.2.4",
+    "eslint": "^7.26.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.23.2",
```

Comment:
- [ ] Ensure you use the same eslint config [rules](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) shared by other students. To update the rules consider raising a PR.